cli/commands/benchmark-url.mjs

```diff
@@ -210,44 +220,37 @@ function outputResults( opt, results ) {
 		}
 	}
 
-	const newRow = ( title ) => {
-		const line = new Array( len + 1 ).fill( '' );
-		line[ 0 ] = title;
-		return line;
-	};
-
 	const headings = [
-		'',
+		'URL',
 		'Success Rate',
 		'Response Time',
+		...Object.keys( allMetricNames ),
 	];
 
-	Object.keys( allMetricNames ).forEach( ( name ) => {
-		headings.push( name );
-	} );
-
 	const tableData = [];
+
 	for ( let i = 0; i < len; i++ ) {
 		const [ url, completeRequests, responseTimes, metrics ] = results[ i ];
 		const completionRate = round(
 			( 100 * completeRequests ) / ( opt.number || 1 ),
 			1
 		);
 
-		const tableRow = [
-			url,
-			`${ completionRate }%`,
-			round( calcMedian( responseTimes ), 2 ),
-		];
-
+		const vals = { ...allMetricNames };
 		for ( const metric of Object.keys( metrics ) ) {
-			const metricAvgMs = round( calcMedian( metrics[ metric ] ), 2 );
-			tableRow.push( metricAvgMs );
+			vals[ metric ] = `${ round( calcMedian( metrics[ metric ] ), 2 ) }`;
 		}
 
-		tableData.push( tableRow );
+		tableData.push( [
+			url,
+			`${ completionRate }%`,
+			round( calcMedian( responseTimes ), 2 ),
+			...Object.values( vals ),
+		] );
 	}
 
+	console.log( headings	,tableData );
```

Comment:
I think this is from debugging and should be removed?

Reply:
Yeah, this is for debugging, removed 😄